platform/cisco-8000/rules.mk

```diff
@@ -0,0 +1,14 @@
+##############################################################
+## Top level platform rules.mk
+##############################################################
+CISCO_PLATFORM_BRANCH := cisco-8000/master
+CISCO_PLATFORM_URL := https://$(GITUSER):$(GITTOKEN)@Cisco-sonic-8000/platform-cisco-8000.git
+CISCO_PLATFORM_SRC=$(PLATFORM_PATH)/src
```

Comment:
In slave.mk, only include "$(PLATFORM_PATH)/rules.mk", so may need to change the CISCO_PLATFORM_SRC=$(PLATFORM_PATH).
Or another options, you can add platform/cisco-8000/rules.mk, and in the file to include $(PLATFORM_PATH)/src/rules.mk